RetrospectiveExtension.Backend/ReflectHub.cs

```diff
@@ -3,119 +3,159 @@
 using Microsoft.ApplicationInsights.Extensibility;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.SignalR;
+using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using CollaborationStateService.Configuration;
 using static ReflectBackend.ReflectBackendSignals;
 
+
 namespace ReflectBackend
 {
-  public enum ReflectBackendSignals
-  {
-    receiveNewItem,
-    receiveUpdatedItem,
-    receiveUpdatedBoard,
-    receiveDeletedItem,
-    receiveDeletedBoard,
-    receiveNewBoard
-  }
-
-  [Authorize]
-  public class ReflectHub : Hub
-  {
-    private readonly TelemetryConfiguration _config;
-    private readonly TelemetryClient _insights;
-
-    public ReflectHub()
+    public enum ReflectBackendSignals
     {
-      this._config = TelemetryConfiguration.CreateDefault();
-      this._insights = new TelemetryClient(this._config);
+        ReceiveNewItem,
```

Comment:
conventions for enums to be Capital letters for C#

Reply:
@mhassaninmsft We may want to keep `receiveNewItem` since when we use ToString() method to convert it to string below, it matches the exact event name on the front-end side. I think that the front end is subscribed to `receiveNewItem` event name:

https://github.com/NarmathaBala/vsts-extension-retrospectives/blob/master/RetrospectiveExtension.Frontend/dal/reflectBackendService.tsx#L11